libkbfs/mdserver_remote.go

```diff
@@ -385,10 +388,30 @@ func (md *MDServerRemote) cancelObservers() {
 	}
 }
 
+// CancelRegistration implements the MDServer interface for MDServerRemote.
+func (md *MDServerRemote) CancelRegistration(ctx context.Context, id tlf.ID) {
+	md.observerMu.Lock()
+	defer md.observerMu.Unlock()
+	observerChan, ok := md.observers[id]
+	if !ok {
+		// not registered
+		return
+	}
+
+	// signal that we've seen the update
+	md.signalObserverLocked(
+		observerChan, id, errors.New("Registration canceled"))
+	// Setting nil here indicates that the server thinks we're still
+	// registered, though locally no one is listening.
+	md.observers[id] = nil
```

Comment:
How do we know if the server thinks we are still registered? The comment in the interface says "let the MDServer know that we are no longer interested ...". Does the server refer to a different context?